src/components/dialogs/ConfirmDissolution.vue

```diff
@@ -58,45 +58,45 @@ import { Getter } from 'vuex-class'
 @Component({})
 export default class ConfirmDissolution extends Vue {
   // Prop to display the dialog.
-  @Prop() private dialog: boolean
+  @Prop() private readonly dialog: boolean
 
   // Global getters
   @Getter getEntityName!: string
   @Getter getEntityType!: string
   @Getter getDissolutionConfirmationResource!: DissolutionConfirmationResourceIF
 
   /** The entity title to display. */
-  private get entityTitle (): string {
+  get entityTitle (): string {
     return this.getDissolutionConfirmationResource?.entityTitle
   }
 
   /** The entity title to display. */
-  private get subEntityTitle (): string {
-    return this.getDissolutionConfirmationResource?.subTitle
+  get subEntityTitle (): string {
+    return this.getDissolutionConfirmationResource.subTitle
```

Comment:
You removed the `?`, so if getDissolutionConfirmationResource is undefined/null then this getter will fail.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining